testsuite/gnat2goto/lib/python/test_support.py

```diff
@@ -91,16 +98,19 @@ def filter_cbmc_output(cbmc_output):
     lines = cbmc_output.split("\n")
     return "\n".join(filter(cbmc_match, lines))
 
-def prove(cbmcargs="", debug=False):
+def prove(cbmcargs="", main="", debug=False):
     """Call gnat2goto (and cbmc) on all *.adb files from the current directory
 
     PARAMETERS
       none: yet
     """
     for file in ada_body_files():
-        out = process(debug, file, cbmcargs)
-        if debug:
-            print('<<< DEBUG ' + file + ' >>>')
-            print(out)
-            print('<<< END DEBUG ' + file + ' >>>')
-        print(filter_cbmc_output(out))
+        gnat2goto_success = process_gnat2goto(debug, file, main != "" and file != main)
+        if gnat2goto_success == False:
```

Comment:
why not just `if not process_gnat2goto(...)` instead?

Reply:
So that's the syntax, ok.